frontend/views/components/StartInviting.vue

```diff
@@ -0,0 +1,66 @@
+      defs
+        clipPath#clip0
+          path(fill="#fff" d="M0 0h97v100.031H0z")
+</template>
+
+<script>
+import sbp from '~/shared/sbp.js'
+import { LOAD_MODAL } from '@utils/events.js'
+
+export default {
+  name: 'StartInviting',
+  methods: {
+    openModal () {
+      sbp('okTurtles.events/emit', LOAD_MODAL, 'InviteByLink')
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import "../../assets/style/_variables.scss";
+
+.c-card {
+  display: flex;
+}
+
+.c-svg {
+  width: 6.25rem; // BUG: handle better svg size (responsive)
+  margin-right: $spacer;
+
+  @include tablet {
+    margin-right: 2.5rem;
+  }
+}
+
+.c-sprite {
+  display: none;
+}
+</style>
```

Comment:
If you don't need to animate the svg, there is not point to add them in the markup I believe. It can be simple svg (easier to find, update, download, share etc..)

Reply:
_au contraire mon ami_. I asked @mmbotelho and the SVGs colors will change depending on the theme. I'll bumped a _very old issue_ with a proposal to solve this in a future PR: https://github.com/okTurtles/group-income-simple/issues/2#issuecomment-529131996